org.flatpak.EasyEdit.yml

```diff
@@ -0,0 +1,90 @@
+  - --share=network
+  - --device=dri
+  # Add sandbox overrides
+  - --env=ELECTRON_DISABLE_SANDBOX=1
+  - --filesystem=host
+modules:
+  - name: EasyEdit
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+      - mkdir -p /app/bin/resources/usr/share/metainfo/
+      - mkdir -p /app/bin/locales/
+      - cp -r * /app/bin/ 
+      - mv -f /app/bin//usr/share/metainfo/easyedit.appdata.xml /app/bin/resources/usr/share/metainfo/
+      - mv -f /app/bin/*.pak /app/bin/locales/
+      - mv -f /app/bin/locales/resources.pak /app/bin/
+      - mv -f /app/bin/app.asar /app/bin/resources/
+      - mv -f /app/bin/app-update.yml /app/bin/resources/
+      - cp -r /app/bin/easyedit.png /app/bin/resources/
+      - chmod 755 /app/bin/easyedit
+      - mkdir -p /app/share/applications
+      - mkdir -p /app/share/icons/hicolor/256x256/apps
+      - mkdir -p /app/share/metainfo
+      - rm -f /app/share/applications/org.flatpak.EasyEdit.desktop
+      - cp /app/bin/easyedit.png /app/share/icons/hicolor/256x256/apps/org.flatpak.EasyEdit.png
+      # Remove sandbox dependency
+      - rm -f /app/bin/chrome-sandbox
+      - |
+        cat > /app/share/metainfo/org.flatpak.EasyEdit.appdata.xml <<'EOF'
+        <?xml version="1.0" encoding="UTF-8"?>
+        <component type="desktop-application">
+            <id>org.flatpak.EasyEdit</id>
+            <name>EasyEdit</name>
+            <summary>Simple text editor</summary>
+            <metadata_license>MIT</metadata_license>
+            <project_license>MIT</project_license>
+            <description>
+                <p>A simple text editor for everyday use</p>
+            </description>
+            <screenshots>
+              <screenshot type="default">
+                <image>https://raw.githubusercontent.com/gcclinux/EasyEdit/refs/heads/main/screenshots/sample011.png</image>
+                <caption>PreviewPanel</caption>
+              </screenshot>
+              <screenshot type="default">
+                <image>https://raw.githubusercontent.com/gcclinux/EasyEdit/refs/heads/main/screenshots/sample010.png</image>
+                <caption>SidebySide</caption>
+                </screenshot>
+            </screenshots>
+            <developer_name>Ricardo Wagemaker</developer_name>
+            <url type="homepage">https://github.com/gcclinux/EasyEdit</url>
+            <releases>
+                <release version="1.3.5" date="2024-03-19"/>
+            </releases>
+        </component>
+        EOF
+      - |
+        cat > /app/share/applications/org.flatpak.EasyEdit.desktop <<'EOF'
+        [Desktop Entry]
+        Name=EasyEdit
+        Comment=Simple text editor
+        Exec=easyedit
+        Icon=org.flatpak.EasyEdit
+        Terminal=false
+        Type=Application
+        Categories=Utility;TextEditor;
+        EOF
+    sources:
+      - type: archive
+        url: https://github.com/gcclinux/EasyEdit/releases/download/binary/easyEdit-linux-x86_64.zip
```

Comment:
why isn't this built from source?

Reply:
Because I could not get it to work i use electron-builder and for some reason it keeps failing so decided to build it, zip it and then they to build a flatpak separately.